From c688b28287b1b965c0dc165973e189f1ac28b6f3 Mon Sep 17 00:00:00 2001
From: Ulf Samuelsson <ulf.samuelsson@atmel.com>
Date: Sat, 16 Apr 2011 11:54:21 +0200
Subject: [PATCH 21/24] at91: Clean up choiuce of sam9m10/g45 boards

Signed-off-by: Ulf Samuelsson <ulf.samuelsson@atmel.com>
---
 arch/arm/mach-at91/Kconfig |   32 ++++++++++++--------------------
 1 files changed, 12 insertions(+), 20 deletions(-)

diff --git a/arch/arm/mach-at91/Kconfig b/arch/arm/mach-at91/Kconfig
index 65f59c7..e2fbfae 100644
--- a/arch/arm/mach-at91/Kconfig
+++ b/arch/arm/mach-at91/Kconfig
@@ -498,44 +498,36 @@ endif
 
 # ----------------------------------------------------------
 
-if ARCH_AT91SAM9G45
+if ARCH_AT91SAM9G45 || ARCH_AT91SAM9M10
 
-comment "AT91SAM9G45 Board Type"
+choice 
+	prompt "AT91SAM9G45/M10 Board Type"
 
-config MACH_AT91SAM9M10G45EK
+config MACH_AT91SAM9G45EKES
 	bool "Atmel AT91SAM9G45-EKES Evaluation Kits"
 	select HAVE_NAND_ATMEL_BUSWIDTH_16
+	depends on ARCH_AT91SAM9G45
 	help
 	  Select this if you are using Atmel's AT91SAM9G45-EKES Evaluation Kit.
 	  "ES" at the end of the name means that this board is an
 	  Engineering Sample.
 
-endif
-
-# ----------------------------------------------------------
-
-if ARCH_AT91SAM9M10
-
-comment "AT91SAM9M10 Board Type"
-
-choice 
-	prompt "Atmel Board"
+config MACH_AT91SAM9M10G45EK
+	bool "Atmel AT91SAM9M10G45-EK Evaluation Kits"
+	select HAVE_NAND_ATMEL_BUSWIDTH_16
+	depends on ARCH_AT91SAM9G45 || ARCH_AT91SAM9M10
+	help
+	  Select this if you are using Atmel's AT91SAM9M10G45-EK Evaluation Kit.
 
 config MACH_AT91SAM9M10EKES
         bool "Atmel AT91SAM9M10-EKES Evaluation Kit"
 	select HAVE_NAND_ATMEL_BUSWIDTH_16
+	depends on ARCH_AT91SAM9M10
         help
           Select this if you are using Atmel's AT91SAM9M10-EKES Evaluation Kit.
           "ES" at the end of the name means that this board is an
           Engineering Sample.
 
-config MACH_AT91SAM9M10G45EK
-        bool "Atmel AT91SAM9M10G45-EK Evaluation Kit"
-	select HAVE_NAND_ATMEL_BUSWIDTH_16
-        help
-          Select this if you are using Atmel's AT91SAM9M10G45-EK Evaluation Kit.
-	  <http://atmel.com/dyn/products/tools_card_v2.asp?tool_id=4735>
-
 endchoice
 
 endif
-- 
1.6.3.3

