From d4473b1414b9ca8ed240f24eb4810adab1fc02c2 Mon Sep 17 00:00:00 2001
From: Ulf Samuelsson <ulf.samuelsson@atmel.com>
Date: Mon, 17 Jan 2011 00:01:53 +0100
Subject: [PATCH 09/13] Fix bad parameters when loading image

---
 main.c |   20 ++++++++++++--------
 1 files changed, 12 insertions(+), 8 deletions(-)

diff --git a/main.c b/main.c
index 78dac44..9ca56ac 100644
--- a/main.c
+++ b/main.c
@@ -37,6 +37,7 @@
 #include "dataflash.h"
 #include "nandflash.h"
 #include "flash.h"
+#include "gpio.h"
 #ifdef CONFIG_USER_HW_INIT
 void user_hw_init(void);
 #endif
@@ -51,6 +52,9 @@ void LoadLinux();
 
 void LoadWince();
 
+unsigned int	img_address	= IMG_ADDRESS;
+unsigned int	img_size	= IMG_SIZE;
+
 /*------------------------------------------------------------------------------*/
 /* Function Name       : main							*/
 /* Object              : Main function						*/
@@ -59,9 +63,9 @@ void LoadWince();
 /*------------------------------------------------------------------------------*/
 int main(void)
 {
-unsigned int	img_address	= IMG_ADDRESS;
-unsigned int	img_size	= IMG_SIZE;
-unsigned int	jump_addr	= JUMP_ADDR;
+
+
+// unsigned int	jump_addr	= JUMP_ADDR;
 
     /*
      * ================== 1st step: Hardware Initialization ================= 
@@ -86,9 +90,9 @@ unsigned int	jump_addr	= JUMP_ADDR;
 
 #if defined(CONFIG_DUAL_BOOT)
 	if(alternate_boot_button()) {
-		dbg_print_cr(">Alternate image");
+		dbgu_print(">Alternate image\r\n");
 		img_address	= ALT_IMG_ADDRESS;
-		img_address	= ALT_IMG_SIZE;
+		img_size	= ALT_IMG_SIZE;
 	} else {
 #else
 	{
@@ -101,20 +105,20 @@ unsigned int	jump_addr	= JUMP_ADDR;
 #else
 
 #if defined(CONFIG_DATAFLASH) || defined(CONFIG_DATAFLASH_CARD)
-    load_df(AT91C_SPI_PCS_DATAFLASH, img_address, img_address, JUMP_ADDR);
+    load_df(AT91C_SPI_PCS_DATAFLASH, img_address, img_size, JUMP_ADDR);
 #endif
     /*
      * Load from Nandflash in RAM 
      */
 #if defined(CONFIG_NANDFLASH)
     read_nandflash((unsigned char *)JUMP_ADDR, (unsigned long)img_address,
-                   (int)img_address);
+                   (int)img_size);
 #endif
     /*
      * Load from Norflash in RAM 
      */
 #ifdef CONFIG_FLASH
-    load_norflash(img_address, img_address, JUMP_ADDR);
+    load_norflash(img_address, img_size, JUMP_ADDR);
 #endif
 #if defined(CONFIG_SDCARD)
     load_SDCard();
-- 
1.7.1

