From 45efda86cd1539292edb9da08db53553505af6f3 Mon Sep 17 00:00:00 2001
From: Ulf Samuelsson <ulf.samuelsson@atmel.com>
Date: Mon, 17 Jan 2011 00:04:51 +0100
Subject: [PATCH 10/13] Update MAKEALL script

---
 MAKEALL |  114 +++++++++++++++++++++++++++++++++++++++------------------------
 1 files changed, 71 insertions(+), 43 deletions(-)

diff --git a/MAKEALL b/MAKEALL
index b0eb73b..95bbc37 100755
--- a/MAKEALL
+++ b/MAKEALL
@@ -1,69 +1,97 @@
 #!/bin/sh
-rm -f binaries/*
-rm -rf log
-rm -fr result
-mkdir -p log/FAIL
-mkdir result
+rm	-f	binaries/*
+rm	-rf	log
+rm	-fr	result
+mkdir	-p	log
+mkdir	-p	result/FAIL
+mkdir	-p	result/OK
+mkdir	-p	result/binaries
+mkdir	-p	result
+mkdir	-p	result/map
+mkdir	-p	result/elf
 skip=0
 
 function build()
 {
 	make mrproper
 	make $1_defconfig
-	make > log/$1.log 2>&1 || mv log/$1.log log/FAIL/$1.log.FAIL
-	mv binaries/* result/
-	mv log/* result/
+	make > log/$1.log 2>&1 || mv log/$1.log result/FAIL/$1.log.FAIL
+	mv log/$1.log result/OK/$1.log.OK	|| echo
+	mv binaries/*.elf	result/elf
+	mv binaries/*.map	result/map
+	mv binaries/*.bin	result/binaries
 }
 
-#build	at91cap9adk
-#build	at91cap9df
-#build	at91cap9f
+build	at91sam9m10sd_android
+build	at91sam9m10sdu
+build	at91sam9m10ek
+build	at91sam9m10df
+build	at91sam9m10sd
+build	at91sam9m10nf
 
-#build	at91cap9stk
-#build	at91cap9stkdf
-#build	at91cap9stkf
+build	afeb9260
 
-build	at91sam9260df
-build	at91sam9260nf
+build	at91sam9m10nfes
+build	at91sam9m10sdes
+build	at91sam9m10ekes
+build	at91sam9m10dfes
 
-build	at91sam9g20df
-build	at91sam9g20sd
-build	at91sam9g20nf
-
-
-build	at91sam9261df
-build	at91sam9261nf
+build	at91sam9xeek
+build	at91sam9xedf
+build	at91sam9xedfc
+build	at91sam9xenf
 
+build	at91sam9g10ek
+build	at91sam9g10nf
 build	at91sam9g10df
 build	at91sam9g10sd
-build	at91sam9g10nf
-
-build	at91sam9rldf
-build	at91sam9rlsd
-build	at91sam9rlnf
-
-build	at91sam9263df
-build	at91sam9263nf
 
-build	at91sam9g45dfes
+build	at91sam9g45sdes_android
 build	at91sam9g45sdes
+build	at91sam9g45ekes
+build	at91sam9g45dfes
 build	at91sam9g45nfes
 
+build	at91cap9stkdf
+build	at91cap9stkf
+build	at91cap9stk
+build	at91cap9f
+build	at91cap9df
+build	at91cap9adk
+
+build	at91sam9260nf
+build	at91sam9260dfc
+build	at91sam9260df
+build	at91sam9260ek
+
+build	at91sam9g20nf
+build	at91sam9g20dfc
+build	at91sam9g20sd
+build	at91sam9g20df
+build	at91sam9g20ek
+
+build	at91sam9261dfc
+build	at91sam9261nf
+build	at91sam9261ek
+build	at91sam9261df
+
+build	at91sam9263ek
+build	at91sam9263nf
+build	at91sam9263df
+build	at91sam9263dfc
+
+build	at91sam9g45ek
 build	at91sam9g45df
-build	at91sam9g45sd
 build	at91sam9g45nf
+build	at91sam9g45sd
 
-build	at91sam9m10df
-build	at91sam9m10sd
-build	at91sam9m10nf
-
-build	at91sam9xedf
-build	at91sam9xenf
-#build	at91sam9xedfc
-#build	at91sam9xeek
+build	at91sam9rlnf
+build	at91sam9rldf
+build	at91sam9rlsd
+build	at91sam9rlek
 
-rm -fr binaries
-rm -fr log
+#rm -fr binaries
+#rm -fr log
 echo
 echo
 echo "### Done!"
-- 
1.7.1

